src/libshadertrap/src/parser.cc

```diff
@@ -254,15 +257,83 @@ bool Parser::ParseCommandAssertEqual() {
               }
               argument_identifier_2 = std::move(token);
               return true;
+            }},
+
+           {Token::Type::kKeywordFormat,
+            [this, &format_entries, &start_token]() -> bool {
+              bool seen_correct_identifier = false;
```

Comment:
Change the name to "seen_at_least_one_format_entry"

Reply:
Done!